Sim/SimPapas/src/PapasBuildBlocksTool.cpp

```diff
@@ -21,8 +20,9 @@ StatusCode PapasBuildBlocksTool::clear() {
   return StatusCode::SUCCESS;
 }
 
-StatusCode PapasBuildBlocksTool::run(papas::Event& pevent) {
+StatusCode PapasBuildBlocksTool::run(papas::Event& pevent, std::shared_ptr<papas::Detector> det) {
   auto history = papas::Nodes();
+  (void)det;  // avoid warninf
```

Comment:
Could you please give me more information about this line? why it's there and what it does? 
Also, please improve the comment

Reply:
The blockbuilder tol uses the IPapasTool interface and this has an optional argument which is the papasdetector. In the case of the Blockbuilder tool the papas detector " det" is not used. This generates a warning message.  (void)det is a trick to suppress the warning message.

Reply:
Still, I think the comment should be developped and improved a bit. most people will not know what is "warninf" (or maybe do you mean warning??) if yes, just say write which warning.